comp/process/bundle.go

```diff
@@ -47,6 +49,9 @@ func Bundle() fxutil.BundleOptions {
 
 		hostinfoimpl.Module(),
 		expvarsimpl.Module(),
+		statusimpl.Module(),
+		coreStatusImpl.Module(),
```

Comment:
I thought it makes sense to add them as part of the bundle. Especially since the  apiserver now requires the status component